src/components/Extensibility/components-form/StringRenderer.js

```diff
@@ -16,11 +20,31 @@ export function StringRenderer({
   required,
   compact,
   placeholder,
+  originalResource,
   ...props
 }) {
+  const { itemVars } = useVariables();
+  const jsonata = useJsonata({
+    resource: originalResource,
+    scope: value,
+    value,
+  });
+
   const { tFromStoreKeys, t: tExt, exists } = useGetTranslation();
   const schemaPlaceholder = schema.get('placeholder');
   const readOnly = schema.get('readOnly') ?? false;
+  const [decoded] = jsonata(
+    schema.get('decoded'),
+    itemVars(originalResource, schema.get('itemVars'), storeKeys),
+  );
+
+  if (decoded) {
+    try {
+      value = base64Decode(value);
+    } catch (e) {
+      // noop
```

Comment:
![image](https://user-images.githubusercontent.com/46759954/212889588-4fe2f39c-8364-4e8e-ad2a-4063a1394478.png)

Reply:
we have to catch the error, but we don't want to do anything, thus the noop